src/services/billing/cost-management/cost-dashboard/modules/CostDashboardPeriodSelectDropdown.vue

```diff
@@ -112,7 +130,23 @@ export default {
             state.customRangeModalVisible = false;
         };
 
-        watch(() => props.fixedPeriod, () => {
+        const handleSelectedFixDate = async (isSelected: boolean) => {
+            try {
+                const periodType = isSelected ? 'FIXED' : 'AUTO';
+                const fixedPeriod = isSelected ? state.period : {};
+                await SpaceConnector.client.costAnalysis.dashboard.update({
+                    dashboard_id: props.dashboardId,
+                    period_type: periodType,
+                    period: fixedPeriod,
+                });
+                emit('update:fixed-period', periodType, fixedPeriod);
+            } catch (e) {
+                ErrorHandler.handleError(e);
+            }
+        };
+
+        watch(() => props.fixedPeriod, (after, before) => {
+            if (isEqual(props.fixedPeriod, state.period) || (isObjectEmpty(before) && !isObjectEmpty(after))) return;
```

Comment:
```suggestion
            if (after === state.period || (isObjectEmpty(before) && !isObjectEmpty(after))) return;
```

참조값 비교만 하여도 아래에서 위로 올려준 객체인지 아닌지를 구별 가능하므로, isEqual 로 감싸지 않아도 좋을 것 같습니다.